packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -1105,9 +1238,17 @@ class _TimePickerDialogState extends State<_TimePickerDialog> {
     _selectedTime = widget.initialTime;
   }
 
+  @override
+  void didChangeDependencies() {
+    super.didChangeDependencies();
+    localizations = MaterialLocalizations.of(context);
+    _announceToAccessibility(localizations.timePickerHourModeAnnouncement);
```

Comment:
This means every time any inherited widget that this widget depends on changes, we'll announce this. That doesn't seem right. What if it's, e.g., the TickerMode? Or the Theme, while the theme is animating?

Reply:
Done.